packages/fiori/src/themes/UploadCollectionItem.css

```diff
@@ -114,6 +118,10 @@
 	margin-right: 0.125rem;
 }
 
+.ui5-li-deletebtn {
+	display: inline-block;
```

Comment:
This breaks the alignment when some "selection-mode" is set to the UploadCollection. You can add to the ListItem.css:

**.ui5-li-detailbtn,**
.ui5-li-deletebtn {
	display: flex;
	align-items: center;
}